### DISCUSSION TITLE
Security

### DISCUSSION THREAD

**Discussion by BlackSparkz** [leonlatsch/Photok#191] [Q&A]
What the mean of (technologies like, AES-256 encryption) you explain us well

**Comment by leonlatsch** [comment]
So AES-256 is used for encrypting your photos.

Bcrypt is also used, but for checking your password at login.

Hope this helps.

**Comment by FlorianPicca** [comment]
How do I submit a security issue if I found one ?

**Reply by leonlatsch** [reply]
The way to wo would be to open an issue.
